src/styles/index.css

```diff
@@ -1,9 +1,81 @@
-.next-lib-template_container {
-    background-color: black;
-    color: white;
-
-    &::after {
-        content: '';
-        margin-bottom: 50px;
-    }
+.next-button_button {
```

Comment:
I think i would call this just `.next-button`, since we don't have any other class, adding the button suffix seems redundant 👍

Reply:
Nice catch, this is a leftover as before we had `.next-button_button` and `.next-button_link` classes!

I'll change this 👍